package.json

```diff
@@ -32,6 +32,7 @@
     "cli-table3": "^0.6.3",
     "esbuild": "^0.19.12",
     "eslint": "~8.56.0",
+    "glob": "^10.4.5",
```

Comment:
We have a benchmark for glob libs here [packages/utils/perf/glob](https://github.com/code-pushup/cli/tree/main/packages/utils/perf/glob).

Lets try to use the fastest and the one with least deps

Reply:
I did try `fast-glob`. But for some reason, it wasn't playing nicely with `memfs`, so it was blocking me from unit testing the monorepo handlers properly :disappointed: I couldn't figure out why, in the GitHub Action codebase I was able to test `fast-glob` using `memfs` without any problems (maybe Jest vs Vitest difference). But I spent a few hours trying get it working here without success, then I tried switching to `glob` as the next best option, and at it all started working :man_shrugging: 

I know that `fast-glob` is faster than `glob` in the benchmark, but performance isn't the only factor. The potential milliseconds of runtime difference is outweighted by the hours saved on development time, in my opinion :nerd_face: Otherwise I don't see much difference between them - both have relatively few dependencies and are widely used.

Reply:
I'd propose replacing the blob benchmark example with some synthetic example of some non-trivial math function like Fibonacci or similar. Meaning we could get rid of these unnecessary multiple blob libs deps (could be part of #776 effort)

Reply:
Agreed, let's make that part of the other issue.

Reply:
Created a new issue for it:
- #845